components/VM_Arm/plat_include/ultra96v2/plat/vmlinux.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#define GIC_IRQ_PHANDLE 0x04
```

Comment:
Is there a way to get this from the device tree parser? If this can't be implemented now, a comment would be nice here and in the other files that copies the section from the device tree, so this is a bit more copy/paste save. We have seen on different qemu-arm-virt versions that the value can change - which causes some strange errors then.

Reply:
That would be really nice. We actually had an issue with vpci wasn't working because this value didn't match what was in the rest of the device tree.

This parameter is defined for each platform, so I would track this as a separate issue. But it should be possible to read the gic node to get this value, that way we can avoid needing to define this.